README-TR.md

```diff
@@ -103,18 +103,17 @@ Direkt olarak uygun issuelara referans veren katkı sağlanabilecek arama bağla
 - [İlk defa Açık Kaynağın içine girmek](https://www.nearform.com/blog/getting-into-open-source-for-the-first-time/) yazan [@mcdonnelldean](https://github.com/mcdonnelldean)
-- [Açık Kaynağa Nasıl Katkı Sağlanır](https://opensource.guide/how-to-contribute/) yazan [@GitHub](https://github.com/github)
-- [Kodunuzda Bug'ı Nasıl Bulabilirsiniz?](https://8thlight.com/blog/doug-bradbury/2016/06/29/how-to-find-bug-in-your-code.html) yazan [@dougbradbury](https://twitter.com/dougbradbury)
-- [Markdownda Uzmanlaşmak](https://guides.github.com/features/mastering-markdown/) by [@GitHubGuides](https://guides.github.com/)
-- [İlk Görev: Katkı Sağlayıcı Sayfası](https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29) yazan [@forCrowd](https://github.com/forCrowd)
+- [Açık Kaynağa Nasıl Katkı Sağlanır](https://opensource.guide/how-to-contribute/) yazan [@GitHub](https://github.com/github/opensource.guide)
+- [Kodunuzda Bug'ı Nasıl Bulabilirsiniz?](https://8thlight.com/insights/how-to-find-a-bug-in-your-code) yazan [@dougbradbury](https://twitter.com/dougbradbury)
+- [Markdownda Uzmanlaşmak](https://docs.github.com/features/mastering-markdown/) by [@GitHubGuides](https://docs.github.com/en)
+- [İlk Görev: Katkı Sağlayıcı Sayfası](https://forcrowd.medium.com/first-mission-contributors-page-df24e6e70705#.2v2g0no29) yazan [@forCrowd](https://github.com/forCrowd)
 - [5 Dakika İçerisinde Açık Kaynağa Nasıl Katkı Sağlarsın](https://medium.freecodecamp.org/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a) yazan [@roshanjossey](https://medium.freecodecamp.org/@roshanjossey)
 - [Hacktoberfest 2018: Nasıl ücretsiz t-shirt alırsın — Kodlama Yeni Olsan Dahi](https://medium.freecodecamp.org/hacktoberfest-2018-how-you-can-get-your-free-shirt-even-if-youre-new-to-coding-96080dd0b01b) yazan [@quincylarson](https://medium.freecodecamp.org/@quincylarson)
 - [Açık Kaynak için Acı Kılavuz](https://medium.com/codezillas/a-bitter-guide-to-open-source-a8e3b6a3c1c4) yazan [@ken_wheeler](https://medium.com/@ken_wheeler)
 - [Yeni Geliştiriciler için Adım Adım İlk Defa Açık Kaynağa Katkı Sağlama Kılavuzu](https://hackernoon.com/contributing-to-open-source-the-sharks-are-photoshopped-47e22db1ab86) yazan [@LetaKeane](http://www.letakeane.com/)
 - [Windows üzerinde Adım Adım Git ve GitHub Öğren)](https://medium.com/@ows_ali/be93518e06dc) yazan [@ows-ali](https://medium.com/@ows_ali)
 - [Neden Açık Kaynak ve Nasıl?](https://careerkarma.com/blog/open-source-projects-for-beginners/) by [@james-gallagher](https://careerkarma.com/blog/author/jamesgallagher/)
 - [Açık Kaynağı kullanmaya nasıl başlanır? - By Sayan Chowdhury](https://www.hackerearth.com/getstarted-opensource/)
-- [Hangi Açık Kaynağa Katkıda Bulunmalıyım?](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) by [@kentcdodds](https://github.com/kentcdodds)
```

Comment:
This can be kept and added back in.

Reply:
done https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/e0eb395efa190930813142bbd76c73002b04cada